edifice/engine.py

```diff
@@ -330,6 +330,10 @@ def __init__(self, root:Element, app=None):
         """
         The per-element hooks for use_async().
         """
+        self._is_stopped: bool = False
```

Comment:
I don't think we gain anything by making this private.

```suggestion
        self.is_stopped: bool = False
```